### ISSUE TITLE
Model converted does't recognize image properly

### ISSUE THREAD

**Issue by fanto88** [tensorflow/tensorflow#46345]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- TensorFlow version (or github SHA if from source):2.4.0

Hi all, i'm trying to make this model: [SSD MobileNet V2 FPNLite 320x320](http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_fpnlite_320x320_coco17_tpu-8.tar.gz)
recognizing my face instead of all other objects of coco dataset. I have already did some training to the model and i wanted to test it out even if wasn't working perfectly fine since the total loss was still high, the problem is that the model is not recognizing anything, the output of the recognized objects is always:
![image](https://user-images.githubusercontent.com/10716240/104225249-71493880-5446-11eb-851a-756dd887f1b9.png)
Even if i move the camera around, so i suppose something is wrong with the model itself. 
Can someone give me a feedback about it? 
This is the model: [Model](https://gofile.io/d/N17Jf4)

My application use more models (same model copied and trained with different dataset for different purpose, and they all works, this is the only that doesn't work properly. ) 

I had a problem with another model and someone suggested me to change IMAGE_MEAN and IMAGE_STD from TFLiteObjectDetectionAPIModel to those values
```
private static final float IMAGE_MEAN = 0.0f;
private static final float IMAGE_STD = 1.0f;
```

But this doesn't work. Any more suggestion?

**Comment by fanto88** [comment]
I actually already fixed the problem. The problem was the loss, after a few more hours of training the model started to work as expected, i'm sorry for the useless issue.
